src/widgets/dropdown/template.tpl

```diff
@@ -1,14 +1,44 @@
 <ul class="CDB-Dropdown-list CDB-Text CDB-Size-medium">
   <li class="CDB-Dropdown-item">
-    <button class="CDB-Dropdown-link js-toggle">Toggle widget</button>
+    <button class="CDB-Dropdown-link js-toggleCollapsed">
+      Toggle widget
+        <div>
+        <% if (collapsed) { %>
```

Comment:
I think we tried that and the problem was the same. Inside of an HTML tag (ie: `<div here>`) the template tags would only work if they where inside of some quotes. Otherwise, they'd make the linter fail :(